index.html

```diff
@@ -2030,6 +2030,79 @@ <h4>Presentations Using Derived Credentials</h4>
             </figcaption>
           </figure>
         </section>
+        <section>
+          <h4>Presentations Including Holder Claims</h4>
+          <p>
+In addition to <a>claims</a> from an <a>issuer</a> in the form of
+<a>verifiable credentials</a>, a <a>verifiable presentation</a> MAY include
+<a>claims</a> directly from the <a>holder</a>. These <a>holder</a> <a>claims</a>
+are considered to be self-asserted. If present, these self-asserted
+<a>claims</a> MUST be formatted as a <a>verifiable credential</a> and included
+in the value of the <code>verifiableCredential</code> <a>property</a> of the
+<a>verifiable presentation</a>. These self-asserted claims can be secured
+by their own mechanism like any other <a>verifiable credential</a> or by
+the same mechanism that secures the <a>verifiable presentation</a> in which
+they are included.
```

Comment:
Hmm, I think the MUST here may not be testable -- or it's suggesting that nothing else can ever be said in a VP without using a VC, which I think would be a mistake. Even adding another custom property that references an encrypted VC or a JWT-secured verifiable credential, for example, would be a "claim" and therefore disallowed.

Here's a tweak to address that and to hopefully make it more clear that the holder can act like any other issuer -- rather than treating them as a special case. I think what's special here is not that the holder is not an issuer (they are), but that because they are the one presenting they may optionally choose to reuse the securing mechanism on the presentation to secure the VCs where they are the issuer. Of course, this is only permitted if the requirements laid out in text following this paragraph are met (`holder` property exists in the VP and has the same value as the `issuer` property in the self-asserted VC).

```suggestion
A <a>holder</a> MAY use the <code>verifiableCredential</code>
<a>property</a> in a <a>verifiable presentation</a> to include
<a>verifiable credentials</a> from any <a>issuer</a>, including
themselves. When the <a>issuer</a> of a <a>verifiable credential</a>
is the <a>holder</a>, its <a>claims</a> are considered to be
self-asserted. These self-asserted claims can be secured by their
own mechanism like any other <a>verifiable credential</a> or by
the same mechanism that secures the <a>verifiable presentation</a>
in which they are included.
```